### PR TITLE
Remove NewsArticle public frontend code 

### PR DESCRIPTION
**Pull request by mgrassotti**
https://trello.com/c/WTk6X44o/626-news-article-migration-final-tasks-6-deploy-2-0-5days

### CODE REVIEW

File: app/controllers/news_articles_controller.rb [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-    set_meta_description(@document.summary)
-  end
-
   private
 
-  def document_class
-    NewsArticle
-  end
-
   def analytics_format
```
Comment (fofr) [inline app/controllers/news_articles_controller.rb]: Does this need to stay? Can the whole file go?

Comment (mgrassotti) [inline app/controllers/news_articles_controller.rb]: 👍 yes you are right

**Comment by mgrassotti** [comment]
yes it was... strange 🤔

**Comment by gpeng** [comment]
Rebased from master as #3109 introduced a little conflict. @bevanloon could you just give this another quick once over?

**Comment by gpeng** [comment]
I've got another PR brewing with a couple of other remnants. I'll remove the route in that.

**Pull request merged**